waiter/docs/parameters.md

```diff
@@ -16,6 +16,7 @@ Additional (optional) parameters that can be set:
 
 |Parameter|Default Value|Valid Values|Description|Guidance|
 |---------|-------------|------------|-----------|--------|
+|`X-Waiter-Allowed-Params`|empty|vector of non-empty strings (token) or comma-separated strings (headers)|The environment variables configured as parameters for a tokenized service via `X-Waiter-Param-*` header which will not trigger an on-the-fly mode.|Expose only environment variables which your command can handle safely in this list. You can also choose to provide default values for these parameters in your `env` configuration.|
```

Comment:
Nitpick: Can you change vector to array here?

Reply:
Done.